packages/dialog/src/mwc-dialog.scss

```diff
@@ -68,3 +74,12 @@ limitations under the License.
     }
   }
 }
+
+.mdc-dialog--normal {
```

Comment:
I would recommend use `:not(.mdc-dialog--popover)` instead of creating a new variant.